doc/decisions/STEPS.md

```diff
@@ -0,0 +1,86 @@
+# Steps
+
+This document describes all steps a decision can run through.
+
+```mermaid
+flowchart LR
+    s((Start)) --> Drafts --> In_Discussion --> In_Progress --> Decided --> Partially_Implemented --> Implemented
+
+    %% Shortcuts:
+    s --> Decided
+    Decided --> Implemented
+
+    %% Dead ends:
+    s --x Rejected
+    s --o Delayed
+    Drafts --x Rejected
+    Drafts --o Delayed
+    In_Discussion --x Rejected
+    In_Discussion --o Delayed
+    In_Progress --x Rejected
+    In_Progress --o Delayed
+```
+
+## Drafts
+
+> This step is recommended if the problem is not yet clear.
+
+The first step is to create a PR with:
+
+- **one** decision, where at least the "Problem" is filled out and "Decision", "Rationale" and "Implications" are **not** yet filled out.
+- a link from [README.md](../README.md) from the "Drafts" section to this decision.
+- optional backlinks from related decisions.
+
+> Everyone must agree that the problem exists so that a decision PR in "Drafts" step can be merged.
+> At least the problem must be clear to everyone involved before the decision can leave the "Drafts" step.
+> It must be so clear that everyone would be able to describe a test case that shows if a solution fixes the problem.
+
+## In Discussion
+
+> This step is recommended if it is not yet clear which solution is the best.
+
+Here you must ensure:
+
+- problem, constraint and assumptions are well-explained and sound
+- consistency with other decisions
+- links from/to related decisions are created
+- there are several considered alternatives, each with rationale and implication
+- "Decision", "Rationale" and "Implications" are **not** yet filled out if there are people arguing for different options
+
+Here the decision should not only have one decision but should describe several solutions.
+For each solution a proposal, rationale and optionally implications should be given.
+
+## In Progress
```

Comment:
How is this different from "In Discussion" when would we use this?

IMO this step should be removed.